File: hw/suarez/U3/HW21TemplateBeverages/src/main/java/ec/edu/espe/cafe/view/BeverageTest.java
package ec.edu.espe.cafe.view;
import ec.edu.espe.cafe.model.Coffee;
import ec.edu.espe.cafe.model.Tea;

/**
 * @author bernardo
 */

public class BeverageTest {

    public static void main(String[] args) {
        Coffee coffee = new Coffee();
        Tea tea = new Tea();
        
        System.out.println("~ MAKING TEA --> \n");
        tea.prepareRecipe();
        System.out.println("Enjoy your tea");
        
        System.out.println("\n~ MAKING COFFEE ---> \n");
        coffee.prepareRecipe();
        System.out.println("Enjoy your coffee");

    }
}

File: hw/paillacho/U3/HW21TemplateMethod/python_version/template/model/beverage_test.py
from coffee import Coffee
from tea import Tea
import sys
import os
sys.path.append(os.path.dirname(os.path.abspath(__file__)))

# Author: Carlos Paillacho


class BeverageTest:

    @staticmethod
    def main():
        tea = Tea()
        coffee = Coffee()

        print("Making Tea ...")
        tea.prepare_recipe()

        print("\nMaking coffee ...")
        coffee.prepare_recipe()


if __name__ == "__main__":
    BeverageTest.main()
